src/main/webapp/WEB-INF/view/activityFeed.jsp

```diff
@@ -0,0 +1,21 @@
+<%@taglib prefix="t" tagdir="/WEB-INF/tags" %>
```

Comment:
This first line tells the JSP where to find the tags, and create a tag library from it, referenced by the variable `t`